crates/client/rpc-core/src/utils.rs

```diff
@@ -116,7 +120,8 @@ pub fn to_deploy_account_tx(tx: BroadcastedDeployAccountTransaction) -> Result<D
         calldata,
         salt: U256::from(contract_address_salt),
         signature,
-        account_class_hash: account_class_hash.into(),
+        account_class_hash: Felt252Wrapper::try_from(&account_class_hash).unwrap(), /* Ok to unwrap, starknet-core
```

Comment:
if you look on top of the file, you'll see that

```
let account_class_hash = tx.class_hash.to_bytes_be();
```

so you may just remove the `.to_bytes_be()`, keep the `FieldElement` and `.into()`